app/controllers/api/questions_controller.rb

```diff
@@ -0,0 +1,43 @@
+class Api::QuestionsController < ApplicationController
+  def index
+    render json: Question.all
+  end
+
+  def show
+    question = Question.find_by(id: params[:id])
+    if question
+      render json: question
+    else
+      render json: {error: "Question does not exist"}
+    end
+
+  end
+
+  def create
+    question = Question.new(question_params)
+
+    if question.save
+      render json: {success: "Question successfully created"}
+    else
+      render json: {error: "Error creating question"}
+    end
+
+  end
+
+  def update
+
+    if Question.update(question_params)
+      render json: {success: "Question successfully update"}
```

Comment:
`Question.update?` you need to find the question and update it

Reply:
Same of the destroy, I can update a question that is not mine. This is not good.